src/config/mongoose_config_parser_toml.erl

```diff
@@ -410,7 +410,13 @@ pool_option([<<"call_timeout">>|_], V) -> [{call_timeout, V}].
 %% path: outgoing_pools.*.connection
 -spec connection_options(path(), toml_section()) -> [option()].
 connection_options([_, _, <<"rdbms">>|_] = Path, Options) ->
-    [{server, rdbms_server(Path, Options)}];
+    {Server, Interval} = rdbms_server(Path, Options),
+    case Interval of
+        undefined ->
+            [{server, Server}];
+        V ->
+            [{server, Server}, {keepalive_interval, V}]
+        end;
```

Comment:
And when it comes to this case of, we could regardless if its value, pass it along. If the default value is `undefined`, we can pass it in place and skip this case-of. @chrzaszcz what do you think it's the cleanest?

Reply:
From the configuration standpoint it would make no difference, but from the equivalence standpoint it would be different thab `cfg` config without such value and because we test it by comparison between `cfg` and `toml`I did it this way

Reply:
For me it is fine as it is now. However, calling `parse_kv` for `keepalive_interval` would make it much easier to plug in a validator for the interval.